### DISCUSSION TITLE
Wallpaper source (Favorites only) is not saved anymore and Favorite Backup is broken

### DISCUSSION THREAD

**Discussion by n4tur4lb0rnch1ll4** [you-apps/WallYou#192] [General]
Hello everyone,

first of all, I've been using the app for about half a year and have been a big fan so far. Until two days ago...

Unfortunately, it now no longer saves the setting that I only want to use my favorites as wallpapers and I also have the problem that it can no longer import my favorites backup - what a shame!

Does anyone know anything about this? Many thanks in advance.

**Comment by Bnyro** [comment]
This has been caused by a bug on the dependency on the Jackson Serialization library unfortunately.

I've moved the app to a different library and released a hot fix at https://github.com/you-apps/WallYou/releases/tag/v0.8.1.

**Reply by n4tur4lb0rnch1ll4** [reply]
Thank you very much, fixed it! Appreciate the effort!

**Comment by Bnyro** [comment]
> Unfortunately, it now no longer saves the setting that I only want to use my favorites as wallpapers and I also have the problem that it can no longer import my favorites backup - what a shame!
>
Unfortunately it turns out that the 8.1 release didn't actually fix importing and exporting backups (see #194) - It will however be fixed with the next release. Sorry for the inconvenience.
